src/mocks/Attachments.json

```
@@ -0,0 +1,41 @@
+[
+    {
+        "entity_id": "1",
+        "file_name": "laser-calibration.txt",
+        "title": "laser-calibration",
+        "description": "Detailed report on the calibration of high-precision lasers used in experiments.",
+        "id": "1",
+        "modified_time": "2024-01-02T13:10:10.000+00:00",
+        "created_time": "2024-01-01T12:00:00.000+00:00"
+    },
+
+    {
+        "entity_id": "1",
+        "file_name": "safety-protocols.pdf",
+        "title": "safety-protocols",
+        "description": "Comprehensive guide outlining safety protocols and procedures for conducting experiments.",
+        "id": "1",
+        "modified_time": "2024-01-02T13:10:10.000+00:00",
+        "created_time": "2024-01-01T12:00:00.000+00:00"
+    },
+
+    {
+        "entity_id": "1",
+        "file_name": "camera-setup-guide.docx",
+        "title": "camera-setup-guide",
+        "description": "Guide providing instructions for setting up and configuring cameras used in experiments.",
```

```suggestion
        "entity_id": "1",
        "file_name": "camera-setup-guide.docx",
        "title": "Camera Setup Guide",
        "description": "Guide providing instructions for setting up and configuring cameras used in experiments.",
```
I think it's more appropriate for the title to be formatted without the hyphens.